main.go

```diff
@@ -63,49 +60,45 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var k8sClusterName string
-	var leaderElectionID string
 	awsCloudConfig := aws.CloudConfig{ThrottleConfig: throttle.NewDefaultServiceOperationsThrottleConfig()}
 	injectConfig := inject.Config{}
+	controllerConfig := config.ControllerConfig{
+		RuntimeConfig: config.RuntimeConfig{
+			Scheme:         scheme,
+			ControllerPort: defaultControllerPort,
```

Comment:
defaultControllerPort can be moved into runtime_config as well

Reply:
ok